<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>275</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GoLearnButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>65</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Учебник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GoTryButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>65</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Демонстратор
виджетов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SettingsBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
